interlok-core/src/main/java/com/adaptris/core/util/DestinationHelper.java

```diff
@@ -118,12 +138,9 @@ public static void logConsumeDestinationWarning(boolean alreadyLogged,
    *
    * @see LoggingHelper#logWarning(boolean, WarningLoggedCallback, String, Object...)
    */
+  @Deprecated
   public static void logWarningIfNotNull(boolean alreadyLogged,
-      WarningLoggedCallback callback, Object d, String text, Object... args) {
-    if (d != null) {
-      LoggingHelper.logWarning(alreadyLogged, callback, text, args);
-    } else {
+      WarningLoggedCallback callback, String text, Object... args) {
       callback.warningLogged();
-    }
   }
```

Comment:
@albinoloverats  I had to fix this on develop, since it's still "useful" possibly; can you merge develop into your PR please.